src/main/java/org/aitesting/passenger/model/Passenger.java

```diff
@@ -0,0 +1,59 @@
+package org.aitesting.passenger.model;
+
+import lombok.Data;
+import org.hibernate.validator.constraints.Email;
+import org.hibernate.validator.constraints.NotBlank;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+@Data
+@Entity
+public class Passenger {
+
+    @Id
+    @NotNull
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @NotBlank
+    private String firstName;
+
+    @NotBlank
+    private String lastName;
+
+    @NotBlank
+    @Email(message = "Please provide a valid e-mail")
+    private String email;
+
+    @NotBlank
+    private String streetAddress;
+
+    @NotBlank
+    private String city;
+
+    @NotBlank
+    private String state;
+
+    @NotBlank
+    private String zip;
+
+    @NotBlank
+    private String phoneNumber;
+
+    public Passenger(){}
+
+    public Passenger(String firstName, String lastName, String email, String streetAddress,
+                     String city, String state, String zip, String phoneNumber){
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.email = email;
+        this.streetAddress = streetAddress;
+        this.city = city;
+        this.state = state;
+        this.zip = zip;
+        this.phoneNumber = phoneNumber;
+    }
```

Comment:
Where are your getters?

Reply:
I am using the project lombok library, it creates the getters and setters at compile time.